addons/web/static/src/core/commands/command_palette.scss

```diff
@@ -1,141 +1,24 @@
 .o_command_palette {
-  margin-top: 10vh;
-
-  &.modal-content {
-    border: none;
-    border-radius: 3px !important;
-    overflow: hidden;
-  }
-
-  > .modal-body {
-    padding: 0;
-    overflow: hidden;
-  }
-
-  &_search {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    background-color: white;
-    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
-    padding: 0.8rem 1em;
-    font-size: 1.5rem;
-
-    .fa-search {
-      color:#C7C7C7;
-    }
-
-    .o_namespace {
-      line-height: 1.8em;
-      min-width: 1em;
-      text-align: center;
-    }
-
-    input {
-      border: none;
-      padding: 0;
-      line-height: 1.8em;
-    }
-  }
-
-  &_listbox {
-    max-height: 50vh;
-    overflow-y: auto;
-    position: relative;
-    scrollbar-width: none;
-    -ms-overflow-style: none;
-
-    &::-webkit-scrollbar {
-      display: none;
+    $-app-icon-size: 1.8rem;
+    
+    > .modal-body {
+        padding: 0;
     }
 
-    &_empty {
-      padding: 1rem 1.3em;
-      line-height: 1.8em;
-      font-style: italic;
-    }
-
-    .o_command_category {
-      padding: 0.5rem 0;
-    }
+    &_listbox {
+        max-height: 50vh;   
+        scrollbar-width: none;
+        -ms-overflow-style: none;
```

Comment:
Those are also "synonyms" for non-Blink browsers like Firefox of:
```css
::-webkit-scrollbar {
  display: none;
}
```

So you can also remove them :wink: